AmericanMedicalAssociation/AMA-style-guide
# AMA-30: adds audience selector directory and template, styles tabs.

https://palantir.atlassian.net/browse/AMA-30

Adds the Audience Selector molecule. Consists of an Audience Selector directory, Audience Selector template, Audience Selector annotation markdown file, and Audience Selector Scss sheet. Styles the tab links.

Steps to test:

1. Navigate to Molecules > Navigation > Audience Selector
2. Observe that Audience Selector is a molecule consisting of a tabbed list with four links.
